client/lost-app/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "lost-app",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+ 
```

Comment:
this proxy was removed because the localhost was opening into 3000 and this was overriding it into not running